src/polyswarm_api/endpoint.py

```diff
@@ -269,38 +269,49 @@ def search_metadata(self, query):
             result_parser=resources.Metadata,
         )
 
-    def submit(self, artifact, artifact_name, artifact_type):
+    def submit(self, artifact, artifact_name, artifact_type, scan_config=None):
```

Comment:
Could we create a sane URL default during this and make it the default based on the type? Would go a long way to making it less likely users encounter Steve's bug.

Reply:
Sure, what do you think it should be the default?

Reply:
Set to `more-time`